core/modules/editor/operations/text/prefix-lines.js

```diff
@@ -17,8 +17,10 @@ exports["prefix-lines"] = function(event,operation) {
 	operation.cutStart = $tw.utils.findPrecedingLineBreak(operation.text,operation.selStart);
 	// Cut to just past the following line break, or to the end of the text
 	operation.cutEnd = $tw.utils.findFollowingLineBreak(operation.text,operation.selEnd);
+	// paramsObject offers count as a string not number
```

Comment:
The comment text is confusing. ... It doesn't fit to the code. right?

Reply:
It refers to the line below, and I didn't find it confusing. The merged fix has no comment at all, and in addition to what this PR does, [casts `paramsObject.count` to a string before parsing it](https://github.com/Jermolene/TiddlyWiki5/commit/a054d100e73db95071299e92c4321c2aa8e42382#diff-130da145d09ea1c35cea010cf80d0e79860765998846dd294da956ab093b04d5R16) which could be confusing to new contributors:
```diff
 exports["prefix-lines"] = function(event,operation) {
+	var targetCount = parseInt(event.paramObject.count + "",10);
 	// Cut just past the preceding line break, or the start of the text
```
Either way, it's a separate issue (and a very minor one).

This PR can be closed. If the current code needs a comment, it should be added in a new PR IMHO.